src/Classes/ItemListControl.lua

```diff
@@ -90,10 +104,28 @@ function ItemListClass:FindSocketedJewel(jewelId, excludeActiveSpec)
 	return equipTree
 end
 
+function ItemListClass:FindAbyssalJewel(item)
+	if item.base.subType and item.base.subType == "Abyss" then
+	-- Search itemTab's itemSets for reference to this abyssal jewel
+		for _, itemSet in pairs(self.itemsTab.itemSets) do
+			if not itemSet then itemSet = self.itemsTab.activeItemSet end
+			for slotName, slot in pairs(itemSet) do
+				if type(slot) == "table" then
+					if slot.selItemId == item.id then
+						return itemSet.title
+					end
+				end
+			end
+		end
+	end
+	return ""
+end
+
 function ItemListClass:GetRowValue(column, index, itemId)
 	local item = self.itemsTab.items[itemId]
 	if column == 1 then
 		local used = self:FindSocketedJewel(itemId, true) or ""
+		local used = used or self:FindAbyssalJewel(item)
```

Comment:
This can be simplified:
```
local used = self:FindSocketedJewel(itemId, true) or self:FindAbyssalJewel(item)
if used then
```

``FindAbyssalJewel`` should also have the same arguments ``FindSocketedJewel`` does, because we want to filter out the active spec in some situations.

Reply:
agreed, when i first tried it, it was `local used = self:FindSocketedJewel(itemId, true) or self:FindAbyssalJewel(item) or ""` following what was there before and it didn't work.

x